### PR TITLE
Created an About page with routing, and added a gallery slider for #33

### PR DESCRIPTION
**Pull request by infernodragon456**
I added an About Us page with proper routing through the navbar, and then added a gallery slider with some other elements.
Screenshot (61)

Please check the changes and merge the pull request @skmirajulislam . Do let me know if there are any changes, as this is my first time contributing to Open Source!

### CODE REVIEW

**Comment by skmirajulislam** [comment]
@infernodragon456  I would like to kindly request if you could provide additional screenshots showcasing the changes you've made. This will help me review and understand your updates in more detail.

**Comment by infernodragon456** [comment]
> @infernodragon456 I would like to kindly request if you could provide additional screenshots showcasing the changes you've made. This will help me review and understand your updates in more detail.

Do you want pictures of the code or the about page?

**Comment by skmirajulislam** [comment]
> > @infernodragon456 I would like to kindly request if you could provide additional screenshots showcasing the changes you've made. This will help me review and understand your updates in more detail.
> 
> Do you want pictures of the code or the about page?

picture of whole website with your changes, dont forget to mention issue number

**Comment by infernodragon456** [comment]
Here are a few screenshots for fix of issue #33 :
Screenshot (62)
Screenshot (63)
Screenshot (64)

**Comment by infernodragon456** [comment]
Do let me know if you require something else for evaluation @skmirajulislam . Also please assign me appropriate tags before merging.

**Comment by skmirajulislam** [comment]
@infernodragon456 dont worry , wait for the evolution, after the reviews ull get the point !

File: CODE_OF_CONDUCT.md
Comment (skmirajulislam) [inline CODE_OF_CONDUCT.md:1]: @infernodragon456 

I wanted to bring to your attention a concern regarding the changes you made to a file that was part of another contributor's pull request, which had already been accepted by the project admin and mentor.

As per GSSoC and Hacktoberfest guidelines, it is against the rules to alter someone else’s PR after it has been reviewed and approved. Such changes can only be made with explicit approval from the project admin or mentor.

**Review by skmirajulislam — CHANGES_REQUESTED** [review]
Dear @infernodragon456,

I hope you are well. I would like to kindly request that you revert the changes made to the file that was part of another contributor's pull request. As previously mentioned, the PR had already been reviewed and accepted by the project admin and mentor, and the modifications were made without prior approval, which goes against GSSoC and Hacktoberfest guidelines.

File: CODE_OF_CONDUCT.md
Comment (infernodragon456) [reply CODE_OF_CONDUCT.md:1]: I notice that I accidentally deleted the contents of CODE_OF_CONDUCT.md while making sure my PR was mergable with main. As of now, this is the status of my repo:
image

As I am new to open source, I do not know how to resolve this issue. If you can, please guide me through the next steps I should take.

**Comment by infernodragon456** [comment]
Should I get an updated fork and make another PR @skmirajulislam ?

**Comment by skmirajulislam** [comment]
@infernodragon456 yeh but for this I have to make this one invalid, raise your , another PR by following rules and regulations of Hactoborfest & GSSoC guidelines

File: CODE_OF_CONDUCT.md
Comment (skmirajulislam) [reply CODE_OF_CONDUCT.md:1]: Don't worry, make a fresh PR which contains updated resources by following GSSoC & Hactoborfest guidelines

**Pull request closed without merging**